blocks/library/embed/index.js

```diff
@@ -243,7 +239,7 @@ function getEmbedBlockSettings( { title, icon, category = 'embed', transforms, k
 				return;
 			}
 
-			const embedClassName = classnames( 'wp-block-embed', {
+			const embedClassName = classnames( {
```

Comment:
@jorgefilipecosta For specific embeds, like youtube for example, the tests are looking for both 'wp-block-embed-youtube' and 'wp-block-embed' to be saved. Since you had suggested removing 'wp-block-embed' from the return of the edit function I thought it made sense to have that consistent throughout. I don't know if it is important for the 'wp-block-embed' class to be used for all embeds or not. If not then maybe the test could be updated to reflect the changed output for classes? Otherwise I can revert back to explicitly using 'wp-block-embed' for the class where it is inserted.